.scalafmt.conf

```diff
@@ -1,4 +1,4 @@
-version = 3.7.4
+version = 3.8.3
```

Comment:
For example, we have no idea what this bump would do to the code?

Reply:
The codebase was already formatted before the version bump and since there is no diff in the code it means there was no breaking changes.

I bumped the version because for some reason scythe current version of scalafmt cli didn't run on my laptop, the newest version does (likely some big fix)

Reply:
Since it's going to run in CI machine with its own job, I'd suggest dropping `diff-ref` and check all the files.

Reply:
> Since it's going to run in CI machine with its own job, I'd suggest dropping `diff-ref` and check all the files.

Sure, diff-ref makes a difference in large codebases but this project is not in that state.

I'll do the change when I get home

Reply:
@eed3si9n This is now done